specification/security/resource-manager/Microsoft.Security/preview/2023-01-01-preview/securityOperators.json

```diff
@@ -235,9 +235,6 @@
       "required": true,
       "type": "string",
       "description": "name of the pricing configuration",
-      "pattern": "^[a-zA-Z][a-zA-Z0-9_]+$",
```

Comment:
Removed validation due to SDK generation failure: (Fatal/DuplicateParameterCollision): Duplicated global non-identical parameter definitions